sql/Pg-database.sql

```diff
@@ -271,9 +271,11 @@ INSERT INTO country(short_name,name) VALUES ('BG','Bulgaria');
 INSERT INTO country(short_name,name) VALUES ('BH','Bahrain');
 INSERT INTO country(short_name,name) VALUES ('BI','Burundi');
 INSERT INTO country(short_name,name) VALUES ('BJ','Benin');
+INSERT INTO country(short_name,name) VALUES ('BL','St. Barthelemy');
```

Comment:
With the introduction of the `sql/changes/` schema management, I consider this file frozen. Even more, I think we should try and remove all data initialization from the schema creation file, because, well, people way well want to *just* create the schema (or *just* load the data into the schema)...
The step to remove the data from the schema isn't the scope of this change, but *not* changing the schema file is probably a good thing. We should add the above explanation to the top of the schema file to help other developers.